pkg/toggle/cache.go

```diff
@@ -0,0 +1,49 @@
+package hyphen
```

Comment:
Lets use something like gocache with the in memory cache. This is what we do in horizon.